### PR TITLE
Make zenburn numbers blue (according to official specs)

### PR DESCRIPTION
**Pull request by ianpan870102**
More info: https://en.wikipedia.org/wiki/Wikipedia:Zenburn

### CODE REVIEW

File: zenburn-theme.el [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ;; Author: Bozhidar Batsov <bozhidar@batsov.com>
 ;; URL: http://github.com/bbatsov/zenburn-emacs
+;; Package-Version: 20200305.737
```
Comment (bbatsov) [inline zenburn-theme.el:7]: I think you modified this by mistake.

Comment (ianpan870102) [reply zenburn-theme.el:7]: You're right, I did, let me change it back.

**Comment by ianpan870102** [comment]
I just removed the package version comment. Please review again, cheers! :-)

**Comment by ianpan870102** [comment]
@bbatsov friendly ping

**Pull request merged**